target/linux/lantiq/dts/DM200.dts

```diff
@@ -0,0 +1,208 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+
+    model = "DM200 - Netgear DM200";
+
+    chosen {
+        bootargs = "console=ttyLTQ0,115200";
+
+    };
+
+    aliases {
+        led-boot = &power_green;
+        led-failsafe = &power_amber;
+        led-running = &power_green;
+
+        led-internet = &dsl_green;
+        led-dsl = &dsl_amber;
```

Comment:
please drop this line. As far as I can see, the dsl LED is a dual colour LED and there are no distinct LEDs for internet and dsl state.